src/pointer.rs

```diff
@@ -42,6 +42,7 @@ impl From<ButtonSource> for PointerButton {
             ButtonSource::Mouse(mouse_button) => PointerButton::Mouse(mouse_button.into()),
             ButtonSource::Touch { finger_id, force } => PointerButton::Touch { finger_id, force },
             ButtonSource::Unknown(n) => PointerButton::Unknown(n),
+            ButtonSource::TabletTool { .. } => todo!("do we want this?"),
```

Comment:
we should remove the todo, but this will be handled with the switch to ui-events